cloudformation-template.yaml

```diff
@@ -1718,10 +1718,29 @@ Resources:
       SecretString:
         Ref: SentryEmailPassword
 
+  S3KmsKey:
+    Type: AWS::KMS::Key
+    Properties:
+      KeyPolicy:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Action: kms:*
+            Resource: "*"
+            Principal:
+              AWS:
+                - Fn::Sub: arn:aws:iam::${AWS::AccountId}:root
+
   SentryBucket:
     Type: AWS::S3::Bucket
     DeletionPolicy: Retain
     Properties:
+      BucketEncryption:
+        ServerSideEncryptionConfiguration:
+        - ServerSideEncryptionByDefault:
```

Comment:
In order for the encryption to actually take place automatically, you have to add another property as per [AWS::S3::Bucket ServerSideEncryptionRule Documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket-serversideencryptionrule.html)

Correct version is something along the lines of:

```
      BucketEncryption:
        ServerSideEncryptionConfiguration:
          - BucketKeyEnabled: true
            ServerSideEncryptionByDefault:
              KMSMasterKeyID:
                Fn::GetAtt: S3KmsKey.KeyId
              SSEAlgorithm: aws:kms
```

Reply:
Got you. Adding changes...

Reply:
Hopefully fixed with https://github.com/Rungutan/sentry-fargate-cf-stack/pull/18/commits/5444a2c86057c46ee01477bb2a9bfd234eac675e